src/depot_gui.cpp

```diff
@@ -974,6 +974,15 @@ struct DepotWindow : Window {
 		}
 	}
 
+	bool last_state;
```

Comment:
Could probably use a comment somewhere in here (either on `last_state` or inside `OnMouseLoop()` explaining why we mark the depot GUI when shift and control are pressed or released.